app/models/flood_risk_engine/comment.rb

```diff
@@ -0,0 +1,6 @@
+module FloodRiskEngine
+  class Comment < ActiveRecord::Base
+    belongs_to :commentable, polymorphic: true
+    belongs_to(:user) if defined?(User)
```

Comment:
why is the if needed and is this a good way to modulate access  ? would't any access to the behaviours mixed in, also have to be guarded in a similar way ?